### ISSUE TITLE
Addition of some CSS3 properties

### ISSUE THREAD

**Issue by mauzertim** [BerndGabriel/HtmlViewer#206]
Is there any plans on addition of some actively used CSS3 properties like border-radius, text-shadow, box-shadow? Additional of such properties would allow to create a modern UI with HtmlViewer.

**Comment by BerndGabriel** [comment]
No, currently there are no plans to add 'eye candy' properties.
